src/main/java/gov/usgs/locator/LocMain.java

```diff
@@ -320,59 +365,116 @@ public static String getCurrentLocalDateTimeStamp() {
     return LocalDateTime.now().format(DateTimeFormatter.ofPattern("yyyyMMdd_HHmmss"));
   }
 
+  /**
+   * This function loads a the contents of a file into a json object
+   *
+   * @param filePath A String containing the full path to the input file
+   * @return A JSONObject containing the file contents, or null if the file was invalid.
+   */
+  public static JSONObject loadJSONFromFile(String filePath) {
+    JSONObject fileJSONObject = null;
+    String fileString = loadStringFromFile(filePath);
+
+    if ("".equals(fileString)) {
+      LOGGER.severe("String from file is empty.");
+      return (null);
+    }
+
+    try {
+      // use a parser to convert to a string
+      JSONParser parser = new JSONParser();
+      fileJSONObject = (JSONObject) parser.parse(fileString);
+    } catch (ParseException e) {
+      LOGGER.severe("JSON string parse exception.");
+      LOGGER.severe(e.toString());
+      return (null);
+    }
+
+    return (fileJSONObject);
+  }
+
+  /**
+   * This function loads a the contents of a file into a string
+   *
+   * @param filePath A String containing the full path to the input file
+   * @return A String containing the file contents, or empty string if the file was invalid.
```

Comment:
totally minor, but since it is returning a String object, why no use a null string object if the file was invalid? The loadJSONFromFile method returns a null object in case of invalid or errors, just seems consistency is good, but maybe again you have more meaning in a String with Null vs. empty?